### PR TITLE
Only install monotonic on python2

### PR DESCRIPTION
**Pull request by yazug**
monotonic is a backport of time.monotonic from python3. Only install it
for python2.

### CODE REVIEW

File: eventlet/__init__.py [hunk 1]
```diff
@@ -21,8 +21,12 @@
...
     # Example: gunicorn
     # https://github.com/eventlet/eventlet/issues/401#issuecomment-327500352
-    import monotonic
-    del monotonic
+    try:
+        import monotonic
+        del monotonic
+    except ImportError:
+        import time
+        del time
```
Comment (temoto) [inline eventlet/__init__.py:29]:
```suggestion
        pass
```
`import time` is not useful, right?

File: eventlet/hubs/hub.py [hunk 2]
```diff
@@ -22,7 +22,12 @@ def alarm_signal(seconds):
 import eventlet.hubs
 from eventlet.hubs import timer
 from eventlet.support import greenlets as greenlet, clear_sys_exc_info
-import monotonic
+try:
+    from monotonic import monotonic as now
```
Comment (temoto) [inline eventlet/hubs/hub.py:26]:
```suggestion
    from monotonic import monotonic
```

File: eventlet/hubs/hub.py [hunk 3]
```diff
@@ -22,7 +22,12 @@ def alarm_signal(seconds):
...
+except ImportError:
+    import time
+    now = time.monotonic
```
Comment (temoto) [inline eventlet/hubs/hub.py:29]:
```suggestion
    from time import monotonic
```

File: eventlet/hubs/hub.py [hunk 4]
```diff
@@ -120,7 +125,7 @@ def __init__(self, clock=None):
         self.closed = []
 
         if clock is None:
-            clock = monotonic.monotonic
+            clock = now
```
Comment (temoto) [inline eventlet/hubs/hub.py:128]:
```suggestion
            clock = monotonic
```

**Review by jstasiak — APPROVED** [review]

**Review by temoto — APPROVED** [review]

File: setup.py [hunk 5]
```diff
@@ -18,7 +18,7 @@
         'dnspython >= 1.15.0',
         'enum34;python_version<"3.4"',
         'greenlet >= 0.3',
-        'monotonic >= 1.4',
+        'monotonic >= 1.4;python_version<"3.3"',
```
Comment (temoto) [inline setup.py:21]:
```suggestion
        'monotonic >= 1.4;python_version<"3.5"',
```
Changed in version 3.5: The function is now always available and always system-wide.

**Pull request merged**